share/doc/homebrew/Interesting-Taps-&-Branches.md

```diff
@@ -36,7 +36,7 @@ Homebrew has the capability to add (and remove) multiple taps to your local inst
 
 *   [homebrew/x11](https://github.com/Homebrew/homebrew-x11): Formulae with hard X11 dependencies.
 
-`brew search` looks in these main taps and as well in [Homebrew/homebrew](https://github.com/Homebrew/homebrew). So don't worry about missing stuff. We will add other taps to the search as they become well maintained and popular.
+`brew search` looks in these main taps as well as in [Homebrew/core](https://github.com/Homebrew/homebrew-core). So don't worry about missing stuff. We will add other taps to the search as they become well maintained and popular.
```

Comment:
Thanks for pondering this! I agree that using fixed width too much degrades readability, even more so due to the added background color. Maybe it's best to limit use of backticks to actual code and use some other highlighting for taps or other things that need to stand out from the regular text. Sadly, there's no demibold in Markdown and bold is indeed a bit loud. I guess our best bet for taps is to always link them to their repository, so they stand out because of the link.